src/test/kotlin/net/maizegenetics/phgv2/cli/TestExtension.kt

```diff
@@ -19,7 +19,7 @@ class TestExtension : BeforeAllCallback {
         const val smallSeqInputDir = "data/test/smallseq/"
         const val smallseqLineAFile = "${smallSeqInputDir}LineA.fa"
         const val smallseqLineAMafFile = "${smallSeqInputDir}LineA.maf"
-        const val smallseqLineBFile = "${smallSeqInputDir}LineB.fa"
```

Comment:
Don't we need a constant for the fasta file?

Reply:
If you mean LineA.fa, thats three lines up (smallseqLineAFile).